pinc/LPage.inc

```diff
@@ -155,18 +171,10 @@ function get_available_page($projectid, $proj_state, $pguser, &$err)
         $page_array = get_available_page_array($project, $round, $pguser, $preceding_proofer_restriction);
     }
     if (!$page_array) {
-        $err = _("No more files available for proofreading for this round of the project.");
-        return null;
+        throw new ProjectNoMorePagesException(_("The project has no more pages available for proofreading in this round."));
```

Comment:
Should we re-word this? For the situation where there are some pages available, but not for the particular user because he proofed them in the previous round.

Reply:
Is the message specific to that situation? That is, for the situation where there are still pages, but not pages that are available to the current user? If so, then I'd change it to say "... available to you ...". But unless that's the case, I'd probably leave it.

Reply:
It is the same message for all cases so I'll leave it.